### ISSUE TITLE
Windows Onecore speech slow with NVDA 2017.3 snapshots

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#7377]
Is it possible to have the Windows Onecore speech run any faster?  It can run considerably faster with Narrator.### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by jcsteh** [comment]
Quoting the User Guide, section 12.5, Windows OneCore Voices:

> Please note that the faster rates available with Narrator are not currently available with NVDA. Also, the speed you select in the Windows Settings affects the rate set in NVDA. These are issues we cannot resolve without changes to Windows. We are hopeful that these will be addressed in a future Windows update.
